rancher/charts
# [dev-v2.8][forward-fit]Fix indent metric relabellings in kube-state-metrics

## Issue: 

Forward fit of https://github.com/rancher/charts/pull/3358
https://github.com/rancher/rancher/issues/43402

## Problem

## Solution


## Testing


## Engineering Testing
### Manual Testing


### Automated Testing


## QA Testing Considerations

Installing rancher-monitoring chart with 
- --set global.cattle.clusterId flag set
- valid metric relabellings in `.Values.prometheus.monitor.metricRelabelings` [relabel docs](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#relabel_config)

 and verify it deploys correctly


### Regressions Considerations


## Backporting considerations
